test_run.lua

```diff
@@ -240,7 +240,8 @@ local function grep_log(self, node, what, bytes)
     if file:seek(-bytes, 'SEEK_END') == nil then
         fail("Failed to seek log file")
     end
-    local found, buf
+    local buf, found = nil
```

Comment:
You are set buf to nil explicitly and found implicitly, it is inconsistent. As I see we mostly prefer to omit explicit `nil`.